validation/src/public/index.html

```diff
@@ -0,0 +1,9 @@
+<html>
```

Comment:
nit: to me `public` isn't an obvious name for the place where static files served from the frontent live - I'd find something like `static` more clear. I see you're serving `public` at the path `static` anyway - why not name it that to begin with?

Reply:
I think the `public` directory name is a convention for web apps, but in this case `static` is more explicit -- I'll change to that.